api/server/server.go

```diff
@@ -1071,7 +1071,9 @@ func (s *Server) postContainersStop(eng *engine.Engine, version version.Version,
 	}
 
 	seconds, err := strconv.Atoi(r.Form.Get("t"))
-	if err != nil {
+	if err != nil && err.(*strconv.NumError).Err == strconv.ErrSyntax {
```

Comment:
IMHO we could just ignore error from Atoi cause it's setting seconds to 0 in any case and we ok with this
`seconds, _ := strconv.Atoi(r.Form.Get("t")` is ok  (https://github.com/docker/docker/commit/e41192a3f8cbfbbfecde03f58a3b2be2b1afd836#commitcomment-10831685)